tests/test_create_or_update_jobs.py

```diff
@@ -21,7 +21,7 @@ def disable_github_org_checking(monkeypatch):
 
 
 # Basic smoketest to test the full execution path
-def test_create_or_update_jobs(tmp_work_dir):
+def test_create_or_update_jobs(tmp_work_dir, db):
```

Comment:
`db` seems unrelated?

Reply:
We now query the db for job_request metadata, where as before we just used the `job` instance that. Given that, this change avoid using the default db and gives it its own isolated db for hygiene.